src/Exception/BadConfigValueAlongsideProvider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Exception;
+
+use InvalidArgumentException;
+
+/**
+ * @author Marko Kunic <kunicmarko20@gmail.com>
+ */
+class BadConfigValueAlongsideProvider extends InvalidArgumentException implements SonataException
+{
+    /**
+     * @param string $configuration
+     *
+     * @return self
+     */
+    public static function fromConfiguration($configuration)
```

Comment:
return type `: self` ?

Reply:
It's on `3.x`, we still support PHP 5.6 😢